cms/djangoapps/contentstore/views/item.py

```diff
@@ -1036,7 +1038,12 @@ def _get_release_date(xblock, user=None):
     Returns the release date for the xblock, or None if the release date has never been set.
     """
     # If year of start date is less than 1900 then reset the start date to DEFAULT_START_DATE
-    if xblock.start.year < 1900 and user:
+    try:
+        xblock_start_year = xblock.start.year
```

Comment:
for the oldmongo the ```xblock.start``` eventually call to_json [here](https://github.com/edx/edx-platform/blob/93faba006c627c921a9b12efe47d64f342cd5e06/common/lib/xmodule/xmodule/fields.py#L77) which raises Value Error when accessing ```xblock.start```